spec/base_spec.rb

```diff
@@ -325,4 +328,16 @@ class InvalidNested < CouchbaseOrm::NestedDocument
             end
         end
     end
+
+    describe '.properties_always_exists_in_document' do
+        fit 'Uses NOT VALUED when properties_always_exists_in_document = false' do
```

Comment:
What is that `fit` method ?
```suggestion
        it 'Uses NOT VALUED when properties_always_exists_in_document = false' do
```

Reply:
it's too only run this test